packages/commercetools/theme/components/StoreLocaleSelector.vue

```diff
@@ -13,7 +13,7 @@
       <SfList>
         <SfListItem v-for="store in availableStores" :key="store.id">
           <a
-            href="javascript:void(0)"
+            href="/"
```

Comment:
After this change, will it take user to main page when clicked? If so, is it desired behavior or user should stay on current page?

Reply:
It does the same thing right now but right now, Lighthouse is shouting that the href attribute is not crawlable and that was fixed with this change.